src/cargo/ops/registry.rs

```diff
@@ -281,7 +281,7 @@ pub fn modify_owners(config: &Config, opts: &OwnersOptions) -> CargoResult<()> {
             try!(config.shell().status("Owner", format!("adding `{:?}` to `{}`",
                                                         v, name)));
             try!(registry.add_owners(&name, &v).map_err(|e| {
-                human(format!("failed to add owners: {}", e))
+                human(format!("Failed to add owners to crate {}: {}", name, e))
```

Comment:
Conventionally our errors start with a lowercase letter, so could this stay as `failed`?